src/services/learninghobbies/index.jsx

```
@@ -249,6 +251,289 @@ export const AddLearningHobbies = ({
+                spinner={isLoading ? <Spinner size="sm" /> : null}
+              />
+            </div>
+          </div>
+        )}
+      </Form>
+    </>
+  );
+};
+
+export const DeleteLearningHobbies = ({
+  setDeleteModal,
```

ESLint

[error] 'setDeleteModal' is missing in props validation

(react/prop-types)